pkg/store/proxy.go

```diff
@@ -254,6 +262,14 @@ func (s *ProxyStore) TimeRange() (int64, int64) {
 	return minTime, maxTime
 }
 
+func (s *ProxyStore) TSDBInfos() []infopb.TSDBInfo {
+	infos := make([]infopb.TSDBInfo, 0)
+	for _, store := range s.stores() {
+		infos = append(infos, store.TSDBInfos()...)
```

Comment:
Do we need to filter out `nil` here if a store doesn't support this call?

Reply:
it seems that the spread operator can already handle nils. I added a test as a safeguard to make sure we don't have some weird regression in the future.